analysed-packages/jquery-mousewheel/version-3.1.13/jquery-mousewheel-3.1.13-SPDX2TV.spdx

```diff
@@ -0,0 +1,389 @@
+SPDXID: SPDXRef-item460147
+FileChecksum: SHA1: 172b62411f8523b74016a77bc8d953741aaedf40
+FileChecksum: SHA256: e29f7e41d6ab7b557a4c205fd171569d37c12cf80033d2c0d21809371be6ca69
+FileChecksum: MD5: e6bb792990736bffce02476e31e74647
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/test/scroll.html
+SPDXID: SPDXRef-item460148
+FileChecksum: SHA1: 82c587595018a29589dd49ae7ef575d26bc6b794
+FileChecksum: SHA256: 645b9c2ce6be07a99a5498d494a7441822b96c8e6cb7adbf3cd538b4537a3ef4
+FileChecksum: MD5: 8230c74a1829aa0dc737d635c3b39198
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/.jshintrc
+SPDXID: SPDXRef-item460150
+FileChecksum: SHA1: 55a0386f2821dda7d42c1d4926ada0ec267aa013
+FileChecksum: SHA256: 059994e24bd8187019bbe7f9e90b7820a4953c371cb7422cc28b616d17f69af2
+FileChecksum: MD5: 04cb46f0fa544b9cae9515b09ae7fdaf
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/Gruntfile.js
+SPDXID: SPDXRef-item460151
+FileChecksum: SHA1: 2fb89c41f97a2831629f6c723abe7aaba6ad1521
+FileChecksum: SHA256: 329dbd2164ecf7733b671f7b6eb33c0129c51c99a2dd9d5c8b29b7e4bf2cc997
+FileChecksum: MD5: fad999d7bd01c82686542e69b0e69ad7
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/LICENSE.txt
+SPDXID: SPDXRef-item460152
+FileChecksum: SHA1: 5a6c10fa73503e53508318861f789ccc6df8ee8f
+FileChecksum: SHA256: 1735f52fa954b27e5ad9ed2321080f3bdfdb5d7ccc00b93f6719a0372edb6f87
+FileChecksum: MD5: e4f7c2d5fc49e79dce89c5e67a304591
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: LicenseRef-MIT
+FileCopyrightText: <text> Copyright jQuery Foundation and other contributors </text>
+
+##File
+
+FileName: jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22.zip/jquery-mousewheel-67289b6b2aa0066d7d78a5807f520387135ffb22/README.md
+SPDXID: SPDXRef-item460153
+FileChecksum: SHA1: 2571cd2d32d639e37ece9366064ec219d007acef
+FileChecksum: SHA256: b160b30338414fbbcd05af6b2605f1dc2c1cebf1b1c590566b3c53b4c72759d3
+FileChecksum: MD5: 57f2b61af2cfd990dc070b839f93eafb
+LicenseConcluded: NOASSERTION
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+
+
+##-------------------------
+## License Information
+##-------------------------
+
+
+  
+LicenseID: LicenseRef-CC0-1.0
```

Comment:
Why is the CC0-1.0 license text there? there is no concluded license and no LicenseInfoInFile with CC0-1.0. If this is there because of the license of the document, this is not requird. It has to be removed. There is also no CC0-1.0 in the disclosure document, so I assume it is there because of the license of the document. As said please remove it

Reply:
Done, thanks